action/protocol/staking/vote_reviser.go

```diff
@@ -35,9 +37,34 @@ func (vr *VoteReviser) Revise(csm CandidateStateManager, height uint64) error {
 		if err != nil {
 			return err
 		}
+		if vr.needCorrectCands(height) {
+			if err := vr.correctAliasCands(csm, cands); err != nil {
+				return err
+			}
+		}
 		vr.storeToCache(height, cands)
 	}
-	return vr.flush(height, csm)
+	return vr.flush(csm, height)
+}
+
+func (vr *VoteReviser) correctAliasCands(csm CandidateStateManager, cands CandidateList) error {
+	_, _, owners, err := readCandCenterStateFromStateDB(csm.SM())
```

Comment:
Returning `owners` with corrected votes looks more readable for me

Reply:
return `cands`

Reply:
this is the owners who did register/update candidate actions during the bug period, use it to correct them in the `cands`